stdlib/public/BackDeployConcurrency/TaskLocal.swift

```diff
@@ -212,13 +218,15 @@ public final class TaskLocal<Value: Sendable>: Sendable, CustomStringConvertible
 // ==== ------------------------------------------------------------------------
 
 @available(SwiftStdlib 5.1, *)
+@_alwaysEmitIntoClient
 @_silgen_name("swift_task_localValuePush")
```

Comment:
hope this is right now then @DougGregor ?

Reply:
Should be `@usableFromInline`.